### ISSUE TITLE
Cannot import resource from SRA using accession codes

### ISSUE THREAD

**Issue by envest** [AlexsLemonade/resources-portal#530]
### Context

I tried to add an SRA resource using an SRA accession code. I was able to successfully upload a GEO dataset.

### Problem or idea

I used two different codes: PRJNA632539 associated with an entire project, and SRR11784499 associated with a single sample. Both had issues: An error occurred while importing. AFAIK these are publicly available.

<img width="2560" alt="Screen Shot 2020-10-09 at 2 47 11 PM" src="https://user-images.githubusercontent.com/10382980/95623006-92168200-0a42-11eb-9031-63c56a07bcc0.png">

<img width="2560" alt="Screen Shot 2020-10-09 at 2 47 45 PM" src="https://user-images.githubusercontent.com/10382980/95623021-9a6ebd00-0a42-11eb-9c87-2935184cb3c3.png">

### Solution or next step

Not sure -- seems like several of the resource types are not available yet, so I expect this would be resolved in time with the others. The interface is really slick and inviting :)

**Comment by kurtwheeler** [comment]
I think this might need an `SRP` accession. We could probably add a feature to auto-convert from PRJNA though.

**Comment by envest** [comment]
Yes -- I was able to add with an SRP. If there are some simple upload rules associated with each resource category (like "SRA resources must be SRPs"), that could save on user frustration.

**Comment by kurtwheeler** [comment]
Ok so here are the three EBI URL patterns we need to be able to support:
* https://www.ebi.ac.uk/ena/browser/view/SRP003819
* https://www.ebi.ac.uk/ena/browser/view/PRJNA168994
* https://www.ebi.ac.uk/ena/browser/view/SRR013547

And the NCBI patterns:
* https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE24542
* https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM609241

NCBI uses their own accession codes, so those are the only accession codes we'll need to support in those patterns.

Basically, the user should be able to put in any of the following and have it work:
```
https://www.ebi.ac.uk/ena/browser/view/SRP003819
https://www.ebi.ac.uk/ena/browser/view/PRJNA168994
https://www.ebi.ac.uk/ena/browser/view/SRR013547
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE24542
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM609241
SRP003819
PRJNA168994
SRR013547
GSE24542
GSM609241
```

**Comment by dvenprasad** [comment]
Adding feedback from Anna here

```
Can’t import from SRA
PRJNA578929
https://www.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA578929&o=acc_s%3Aa
SRX9369609
This worked: SRP288716```
